src/Mouf/Database/QueryWriter/ResultSet.php

```diff
@@ -51,12 +51,12 @@ public function current()
     /**
      * @return int
      */
-    public function key()
+    public function key(): mixed
```

Comment:
I am not sure that we should introduce the `mixed` return type as this would require to target newer PHP version. What's your opinin on this ?